frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -33,6 +33,7 @@ export default class Select extends Component {
     onChange: PropTypes.func.isRequired,
     multiple: PropTypes.bool,
     placeholder: PropTypes.string,
+    disabled: PropTypes.bool,
```

Comment:
nitpick: it doesn't hurt to start avoiding negative names, since it can become very confusing quickly: https://ariya.io/2016/04/api-names-and-begging-the-negatives.

But I know, DOM API influence, etc etc ...

Reply:
Yeah, I tend to agree--mainly doing this to match what is expected by the DOM API